api/v1beta2/imagerepository_types.go

```diff
@@ -116,6 +116,7 @@ type ImageRepositorySpec struct {
 }
 
 type ScanResult struct {
+	Revision   string      `json:"revision"`
```

Comment:
This should be optional in v1beta2. When we promote the API to v1 we’ll make it required. Please add a comment about v1 in here so we’ll not forget about it.

Reply:
Fixed, thanks!